feature/office/src/main/java/com/iteneum/office/presentation/ui/Office.kt

```diff
@@ -29,43 +31,55 @@ import com.iteneum.office.R
  * Comment
  */
 @Composable
-fun OfficeUI() {
-
+fun OfficeUI(vm: OfficeViewModel = viewModel()) //Injecting the viewModel
```

Comment:
You need to change the viewModel() for hiltViewModel() cause we will use navigation compose and viewModel() doesn't work on it